modules/account-lib/src/coin/eth/transactionBuilder.ts

```diff
@@ -606,4 +631,8 @@ export class TransactionBuilder extends BaseTransactionBuilder {
   protected set transaction(transaction: Transaction) {
     this._transaction = transaction;
   }
+
+  private static common2v(common: EthereumCommon): string {
```

Comment:
This function name is a little confusing. Can we call it something like `chainIdToFinalV`?
`common.chainId` [is deprecated](https://github.com/ethereumjs/ethereumjs-monorepo/blob/36d4a06a9590644b2cf20262b74bd53570cf20cd/packages/common/src/index.ts#L1035), you may want to use `chainIdNBN` instead.

Reply:
Please add jsdoc too

Reply:
done